packages/datadog-instrumentations/src/express.js

```diff
@@ -69,6 +67,20 @@ addHook({ name: 'express', versions: ['>=4'] }, express => {
   return express
 })
 
+addHook({ name: 'express', versions: ['>=4.0.0 <5.0.0'] }, express => {
```

Comment:
btw, just an opinion kinda thing but i'm pretty sure this also works
```suggestion
addHook({ name: 'express', versions: ['4'] }, express => {
```

Reply:
I prefer keeping `'>=4.0.0 <5.0.0'` as it's more explicit and makes the version range clear to any one reading the code.

Reply:
Not sure I understand this last comment as `4` is a lot easier to read then `>=4.0.0 <5.0.0`. You instantly know that `4` cover only v4.x, but with `>=4.0.0 <5.0.0` you need a second to parse it. The former is definitely easier to read.